src/core/build_simulations.jl

```diff
@@ -77,21 +75,32 @@ function _build_stages(sim_ref::SimulationRef,
 
     system_to_file = get(kwargs, :system_to_file, true)
     mod_stages = Vector{_Stage}(undef, length(stages))
-
-    for (k, v) in stages
-        verbose && @info("Building Stage $(k)")
-        op_mod = OperationModel(DefaultOpModel, v.model, v.sys;
-                                optimizer = v.optimizer,
-                                parameters = true,
-                                verbose = verbose,
-                                kwargs...)
-        stage_path = joinpath(sim_ref.models,"stage_$(k)_model")
+    for (key, stage) in stages
+        verbose && @info("Building Stage $(key)")
```

Comment:
Not an immediate request for change, but I think you should rely on the logging configuration to control what gets logged rather than a boolean that you have to pass everywhere.

Reply:
Ok, this can be addressed when we add the logging capabilities. This should be done for v0.2.0.